tableauserverclient/models/tableau_auth.py

```diff
@@ -1,5 +1,6 @@
 class TableauAuth(object):
-    def __init__(self, username, password, site=None, site_id='', user_id_to_impersonate=None):
+    def __init__(self, username='', password='', site=None, site_id='', user_id_to_impersonate=None, token_name='',
```

Comment:
It's been a few versions, maybe we can safely remove the deprecated site parameter while we're here. (\cc @RussTheAerialist?)